plugins/plugin-svelte/plugin.js

```diff
@@ -17,23 +17,25 @@ module.exports = function plugin(snowpackConfig, {hot: hotOptions, ...sveltePlug
     svelteRollupPlugin({include: '**/node_modules/**', dev: isDev}),
   );
 
-  let svelteOptions;
+  let {config = '.', ...svelteOptions} = sveltePluginOptions || {};
+  let userSvelteOptions;
   let preprocessOptions;
-  // Note(drew): __config is for internal testing use; maybe we should make this public at some point?
-  const userSvelteConfigLoc =
-    sveltePluginOptions.__config || path.join(process.cwd(), 'svelte.config.js');
+
+  const userSvelteConfigLoc = path.resolve(process.cwd(), `${config}/svelte.config.js`);
+  
   if (fs.existsSync(userSvelteConfigLoc)) {
```

Comment:
needed: if I tell you to look for a specific config file, and it doesn't exist, I would want to see an error thrown (or at least logged to console).